Radzen.Blazor/RadzenChart.razor.cs

```diff
@@ -427,6 +427,37 @@ internal async Task DisplayTooltip()
             }
         }
 
+        /// <summary>
+        /// Displays a Tooltip on a chart without user interaction, given a series, and the data associated with it.
+        /// </summary>
+        /// <param name="series"></param>
+        /// <param name="data"></param>
+        /// <exception cref="ArgumentException"></exception>
+        public async Task DisplayTooltipFor(IChartSeries series, object data)
+        {
+            if (!Series.Contains(series))
+            {
+                throw new ArgumentException($"Series:{series.GetTitle()} does not exist in {nameof(this.Series)}");
+            }
+
+            if (IsJSRuntimeAvailable)
+            {
+                Point ttpos = null;
```

Comment:
I think `ttpos` is kind of a cryptic name. Use either `point` or just `position`.